src/pybel/io/mongodb.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""An exporter from PyBEL graphs to a local Mongo database."""
+
+import logging
+from copy import deepcopy
+from typing import Any, List, Mapping, Optional, Tuple
+
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+from pybel import BELGraph, to_sbel
+from pybel.dsl import Entity, Variant
+from pybel.schema import is_valid_edge, is_valid_node
+
+__all__ = [
+    '_entity_to_dict',
+    '_rm_mongo_keys',
+    'to_mongodb',
+    'find_nodes',
+    'get_edges_from_node',
+    'get_edges_from_criteria'
+]
+
+logger = logging.getLogger(__name__)
+
+
+def _entity_to_dict(entity: Entity) -> Mapping[str, Any]:
+    """Input a pybel Entity and return a dict representing it."""
+    new_node = dict(entity)
+    if new_node['function'] in ['Complex', 'Composite']:
```

Comment:
you should use the constants representing this strings in `pybel.constants`:
`pybel.constants.FUNCTION`, `pybel.constants.COMPLEX`, `pybel.constants.COMPOSITE`, `pybel.constants.MEMBERS`, etc.